src/Synapse/Synapse/Models/SynapseAnalyticsManagementClient.cs

```diff
@@ -462,6 +462,47 @@ public List<RestorePoint> ListSqlPoolRestorePoints(string resourceGroupName, str
             }
         }
 
+        public void DeleteSqlPoolRestorePoint(string resourceGroupName, string workspaceName, string sqlPoolName, string sqlPoolRestorePointCreationTime)
+        {
+            try
+            {
+                if (string.IsNullOrEmpty(resourceGroupName))
+                {
+                    resourceGroupName = GetResourceGroupByWorkspaceName(workspaceName);
+                }
+
+                if (!TestSqlPoolRestorePoint(resourceGroupName, workspaceName, sqlPoolName, sqlPoolRestorePointCreationTime))
+                {
+                    throw new InvalidOperationException(string.Format(Properties.Resources.SqlPoolRestorePointDoesNotExist, sqlPoolRestorePointCreationTime));
+                }
+
+                this._synapseManagementClient.SqlPoolRestorePoints.Delete(resourceGroupName, workspaceName, sqlPoolName, sqlPoolRestorePointCreationTime);
+            }
+            catch (ErrorContractException ex)
+            {
+                throw GetSynapseException(ex);
+            }
+        }
+
+        public bool TestSqlPoolRestorePoint(string resourceGroupName, string workspaceName, string sqlPoolName, string sqlPoolRestorePointName)
+        {
+            try
+            {
+                List<RestorePoint> restorePointList = new List<RestorePoint>();
+                restorePointList = this._synapseManagementClient.SqlPoolRestorePoints.List(
+                    resourceGroupName,
+                    workspaceName,
+                    sqlPoolName)
+                    .ToList();
+
+                return (restorePointList.Count != 0) && (restorePointList.Exists(element => element.Name == sqlPoolRestorePointName));
```

Comment:
> (restorePointList.Count != 0) && (restorePointList.Exists(element => element.Name == sqlPoolRestorePointName)); [](start = 22, length = 112)

This is okay. But I have two suggestions:
1. `restorePointList.Any(element => element.Name == sqlPoolRestorePointName);` can be more concise
2. It seems that `this._synapseManagementClient.SqlPoolRestorePoints.Get` will be more straight forward